src/components/form/templates/types.ts

```diff
@@ -12,6 +14,14 @@ export type ObjectFieldProperty = ObjectFieldTemplateProps['properties'][0];
 
 export type ArrayFieldItem = ArrayFieldTemplateProps['items'][0];
 
+export interface LimeObjectFieldTemplateProps extends ObjectFieldTemplateProps {
+    schema: LimeJSONSchema;
+}
+
 export interface Runnable {
     run: (event: any) => void;
 }
+
+interface LimeJSONSchema extends JSONSchema7 {
+    lime?: LimeSchemaOptions;
+}
```

Comment:
There is already an interface named `LimeJSONSchema` defined in `form/widgets/types.ts`, we should be able to reuse that one :slightly_smiling_face:

Reply:
Yeah, I did find that, and copied this from there. I thought we'd probably want to reuse it, but I was a little bit hesitant since it was defined inside `form/widgets/` and also wasn't exported.

I could move that to `form/form.types.ts`, export it, and then import it both here and in `form/ widgets/types.ts`, if that sounds good to you?

Reply:
I think the problem with having it in `form/form.types.ts` is that it will also expose it to the consumers and I'm not sure we want that, since we will also implicitly expose `JSONSchema7`  :thinking: 
It should be ok to just import it from the other file, or perhaps move it to a separate file if you feel that is more appropriate :slightly_smiling_face:

Reply:
⚡